source/module_esolver/opt/opt_DCsrch.cpp

```diff
@@ -0,0 +1,750 @@
+#include "./opt_DCsrch.h"
+
+extern "C"{
+/* Copyright (C) 1996 Jorge J. More'.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
```

Comment:
This source file is using a different license from ABACUS.

Reply:
I have removed this file temporarily. Later it will be replaced.